### PR TITLE
[16.0][mig] product_supplierinfo_archive: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: product_supplierinfo_archive/static/description/index.html [hunk 1]
```diff
@@ -367,7 +367,7 @@ <h1 class="title">Product Supplier Info Archive</h1>
...
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/lgpl-3.0-standalone.html"><img alt="License: LGPL-3" src="https://img.shields.io/badge/licence-LGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/product-attribute/tree/16.0/product_supplierinfo_archive"><img alt="OCA/product-attribute" src="https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_supplierinfo_archive"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/135/15.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (gdgellatly) [inline product_supplierinfo_archive/static/description/index.html:370]: Says this file is autogenerated, I don't much about it tbh, but is referencing v15?

**Review by gdgellatly — APPROVED** [review]
Functionally fine. Just 1 comment, which I do not know the answer to, it could be fine.

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM!

**Review by JordiMForgeFlow — APPROVED** [review]

**Comment by willmakenoise** [comment]
@JordiMForgeFlow Is it possible to merge this PR?

**Comment by JordiMForgeFlow** [comment]
Hi @JordiBForgeFlow

This one is ready to merge :)

**Pull request merged**